lessons/misc/open-data-intro/lesson.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: 'Opening up your data'
+visible: True
+tags:
+  - misc
+  - beginner
+
+---
+
+ - **Authors**: Derek Howard
+ - **Lesson topic**: A guided discussion on opening up your research data
+
+
+## What is open data?
+[“Open data is data that can be freely used, re-used and redistributed by anyone - subject only, at most, to the requirement to attribute and sharealike.”](http://opendatahandbook.org/guide/en/what-is-open-data/)
+
+Specifically:
```

Comment:
Since this whole section is from the above link, maybe this citation should be a bit more explicit.